apps/vessel-history/features/risk-iuu-indicator/risk-iuu-indicator.tsx

```diff
@@ -0,0 +1,78 @@
+import { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { IconButton } from '@globalfishingwatch/ui-components'
+import InfoFieldHistory from 'features/profile/components/InfoFieldHistory'
+import { ValueItem } from 'types'
+import { VesselFieldLabel } from 'types/vessel'
+import { getUniqueHistoryValues } from 'features/vessels/activity/vessels-activity.utils'
+import styles from '../risk-identity-indicator/risk-identity-indicator.module.css'
+
+export interface RiskIuuIndicatorProps {
+  field: VesselFieldLabel
+  hideTMTDate?: boolean
+  history?: ValueItem[]
+  subtitle?: string
+  title: string
+  vesselName: string
+}
+
+export function RiskIuuIndicator({
+  field,
+  hideTMTDate,
+  history,
+  subtitle,
+  title,
+  vesselName,
+}: RiskIuuIndicatorProps) {
+  const [modalOpen, setModalOpen] = useState(false)
+  const { t } = useTranslation()
+  const uniqueValues = useMemo(() => getUniqueHistoryValues(history), [history])
+  const autoSubtitle = useMemo(() => {
+    if (subtitle) return
+    return `(${uniqueValues.join(', ')})`
+  }, [uniqueValues, subtitle])
+  const hasHistory = history.length > 0
+
+  const openModal = useCallback(() => {
+    if (hasHistory) {
+      setModalOpen(true)
+    }
+  }, [hasHistory])
+  const closeModal = useCallback(() => setModalOpen(false), [])
+
+  return (
+    <div className={styles['container']}>
+      <div className={styles.title}>
+        <div>{title}</div>
+        {(!!subtitle || !!autoSubtitle) && (
+          <div className={styles.subtitle}> {subtitle ?? autoSubtitle}</div>
+        )}
+        <IconButton
+          icon={'info'}
+          size="small"
+          className={styles.info}
+          onClick={openModal}
+        ></IconButton>
+      </div>
+      {hasHistory && (
+        <InfoFieldHistory
+          label={field}
+          history={history}
+          isOpen={modalOpen}
+          hideTMTDate={hideTMTDate}
```

Comment:
It's missing the `datesTemplate` here, thus it is showing the `From: ` prefix when viewing this history from the iuu indicator.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/2205831/192533534-365b3a75-7f15-462a-8b14-27a73253dacc.png">